### PR TITLE
Fix for when the subdomain is blank in sign in

### PR DESCRIPTION
**Pull request by cored**

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.2.2
```
Comment (macat) [inline .ruby-version]: can you remove this file?

File: Gemfile.lock [hunk 2]
```diff
@@ -347,6 +347,3 @@ DEPENDENCIES
   title
   unicorn
   webmock
-
-BUNDLED WITH
```
Comment (macat) [inline Gemfile.lock]: please update your bundle.

File: config/database.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ development: &default
   adapter: postgresql
   database: connect_development
   encoding: utf8
-  host: db
+  host: localhost
```
Comment (macat) [inline config/database.yml]: keep this. use docker-compose to run the project.

**Comment by poopoothegorilla** [comment]
+1

File: Gemfile.lock (hunk 2 above)
Comment (bobbytables) [inline Gemfile.lock]: yes.

**Comment by bobbytables** [comment]
Other than the Gemfile.lock, this looks good to me. :+1:

**Comment by bobbytables** [comment]
:+1: after rebase.

**Pull request merged**